astro/src/content/docs/get-started/download-and-install/silent-mode.mdx

```diff
@@ -11,7 +11,7 @@ Silent mode has been available in FusionAuth since version `1.0.0`. This feature
 
 Silent mode serves two purposes that we will cover in the following sections: installation and upgrades. Keep in mind that both of these processes might fail so you should always be ready to intervene and you should always make backups of existing databases.
 
-## Installation
+## What it does
 
 Silent can be used to automatically create the database and schema that FusionAuth uses. It can also grant the correct privileges to the correct database users so that FusionAuth is not connecting to the database as a super user. When enabled, silent mode will perform the following steps:
```

Comment:
```suggestion
Silent mode can be used to automatically create the database and schema that FusionAuth uses. It can also grant the correct privileges to the correct database users so that FusionAuth is not connecting to the database as a super user. When enabled, silent mode will perform the following steps:
```